discovery/file/file_test.go

```diff
@@ -27,11 +27,13 @@ import (
 	"github.com/prometheus/prometheus/config"
 )
 
+const testDir = "fixtures"
+
 func TestFileSD(t *testing.T) {
-	defer os.Remove("fixtures/_test_valid.yml")
-	defer os.Remove("fixtures/_test_valid.json")
-	defer os.Remove("fixtures/_test_invalid_nil.json")
-	defer os.Remove("fixtures/_test_invalid_nil.yml")
+	defer os.Remove(testDir + "/_test_valid.yml")
```

Comment:
How does go on Windows work? Is the slash file separator supported? Or should we use Filepath.Join?

Reply:
yeah seems broken from the first commit , just proves it was never run on windows ;)

but anyway good catch. I made the changes.